### PR TITLE
Deprecation of IPAddressField in Django 1.7, in favor of

### PR DESCRIPTION
**Pull request by claudiay**
GenericIPAddressField.

Needs to support GenericIPAddressField for current version of Django.

### CODE REVIEW

File: django_filters/filterset.py [hunk 1]
```diff
@@ -285,6 +285,9 @@ def __new__(cls, name, bases, attrs):
     models.IPAddressField: {
         'filter_class': CharFilter,
     },
+    models.GenericIPAddressField: {
+        'filter_class': IPFilter,
```
Comment (kevin-brown) [inline django_filters/filterset.py]: Can't you just use `CharFilter` here, like we are for `IPAddressField`?

**Comment by carltongibson** [comment]
@claudiay — Thanks for the PR here.

My current stance it to be a bit sceptical of new filter subclasses. Can you explain to me what we're getting here? Does @kevin-brown not have a point? 

If there is another filter class it will need adding to the docs. (Prizes of appreciation are available for extra efforts there.)

**Comment by suqi** [comment]
I've also run into this problem.

In order to support GenericIPAddressField to fix this, would you mind if I make a **quickfix** pull request ? (I mean just use a CharFilter for GenericIPAddressField)

thanks.

**Comment by carltongibson** [comment]
@suqi Feel free. (It should be all that's needed right?)

**Comment by suqi** [comment]
@carltongibson Yes, filtering IP as a string should be all that's needed now. However, adding a advanced filter for IPv6 like @claudiay did can be an option for the future.

**Comment by carltongibson** [comment]
Closing in favour of #328

**Pull request closed without merging**